tests/test_datasets.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from dem_stitcher.datasets import get_overlapping_dem_tiles
+
+extents = [
+           # This is overlapping with Points/LineString for missing tiles
+           # See https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/issues/89#issuecomment-1399142499
+           [40.0, 36.0, 44.0, 40.0],
+           # Middle of Atlantic
+           [-46, 22, -45, 23]
+          ]
+dem_names = ['glo_90_missing', 'glo_30']
+
+
+@pytest.mark.parametrize('extent, dem_name', zip(extents, dem_names))
+def test_empty_tiles(extent, dem_name):
+    df_tiles = get_overlapping_dem_tiles(extent, dem_name)
```

Comment:
@forrestfwilliams - this is a problem issue you found - thank you.

Reply:
Didn't quite select everything - but the test I am referring to is `test_empty_tiles` and the result should be that the extent found in the bug should now be empty because the intersections are linestrings and/or points.